core/src/main/java/com/radixdlt/api/system/system-api-schema.yaml

```diff
@@ -134,6 +134,36 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  "/db-checkpoint":
+    post:
+      summary: Create a persistent snapshot (checkpoint) of the state manager database
```

Comment:
I think we need to keep `summary` short, down to "Three Capitalized Words", like the other ones:
<img width="306" alt="image" src="https://github.com/radixdlt/babylon-node/assets/119413677/ccfcf116-0927-4cfe-bb0f-08a187d8fa1b">
(tools often want to render it as a list of one-liners)

"Create DB Checkpoint"?